build/yamls/base/crds.yml

```diff
@@ -717,6 +717,13 @@ spec:
                     cidr:
                       type: string
                       format: cidr
+                serviceSelector:
```

Comment:
should this called serviceRef or serviceReference? would it also make sense to use it in conjunction of `namespaceSelector` to select services from matching namespaces, since our group is a cluster scoped one.. i guess we need PM input?

Reply:
Yes that'll be nice. What I'm thinking now is that we can change this to `serviceRef` and keep what the semantics as is. `serviceSelector` used in conjunction with `namespaceSelector` feels a bit weird to me now, as I'm not sure if there actually exists usecases to select multiple services of the same name in multiple namespaces.

Reply:
there are talks of services in  multiple clusters with the same name :) 
also i saw in upstream doc they have used the namespaceSelector in conjunction with the name.. @jianjuns any thoughts ?

Reply:
I feel it does not make much sense to select Services with the same name in multiple Namespaces, but namespaceSelector can be used to select a single Namespace too.
I feel Namespace+name is more convenient to use than Namespace selector + name, but I am not very certain.